### ISSUE TITLE
Can't click right shoulder button.

### ISSUE THREAD

**Issue by xdTAGCLAN** [jrbudda/Vivecraft_116#76]
When I play Vivecraft at the start I can click but then all of a sudden I just can not click anymore. Only in Minecraft. I can still click when in the Steam VR menu.

**Comment by xdTAGCLAN** [comment]
I am on quest 1 btw

**Comment by Techjar** [comment]
Maybe your bindings got messed up in SteamVR.

**Comment by xdTAGCLAN** [comment]
No since it happens everytime I open the game.

**Comment by Techjar** [comment]
Oh you mean it works at the start of each time you run the game. Weird.

To be clear, you're actually playing on 1.16.5 right? Not 1.12.2 or something?

**Comment by xdTAGCLAN** [comment]
![Capture](https://user-images.githubusercontent.com/63149454/115505567-f4660000-a279-11eb-8acb-7cd99cd4e88f.PNG)

**Comment by xdTAGCLAN** [comment]
It always takes from 5 seconds to like 30.

**Comment by Techjar** [comment]
Hmm, this is an odd issue then. You're saying you can't click the world, menus, or anything in the game?

**Comment by xdTAGCLAN** [comment]
If i'm not fast enough i can not join a world yea.

**Comment by Techjar** [comment]
Huh. Unfortunately I haven't a clue what might be going on there. Doesn't look like you've got Forge so it wouldn't be a mod issue either. This kinda has to be something going wrong with SteamVR honestly, but I don't know where to start isolating it as I've never seen this kind of behavior before.

**Comment by Techjar** [comment]
Wait, I just got one idea. Do you have the game in fullscreen? If it's in fullscreen and getting minimized, that fucks the ability to use the mouse.

**Comment by xdTAGCLAN** [comment]
So you mean i should play in fullscreen right?

**Comment by Techjar** [comment]
I mean it's not that you should or shouldn't, either way is fine. What's important is that you don't minimize/alt+tab the game.

**Comment by xdTAGCLAN** [comment]
I don't remember doing that...

**Comment by Techjar** [comment]
Let me put it in simpler terms: is the game visible on your desktop when the issue happens?

**Comment by xdTAGCLAN** [comment]
Ima check.

**Comment by xdTAGCLAN** [comment]
Ah I think that's it! It automatically minimized the game. I kept it open and now it woks!

**Comment by Techjar** [comment]
The window minimization bug strikes again! Something must be stealing focus. Might wanna look into that, or just turn off fullscreen.
